pkg/securitycontext/util.go

```diff
@@ -167,3 +172,48 @@ func AddNoNewPrivileges(sc *v1.SecurityContext) bool {
 	// handle the case where defaultAllowPrivilegeEscalation is false or the user explicitly set allowPrivilegeEscalation to true/false
 	return !*sc.AllowPrivilegeEscalation
 }
+
+var (
+	defaultMaskedPaths = []string{
+		"/proc/kcore",
```

Comment:
Jess, looks like we could add `/proc/acpi` (from https://github.com/moby/moby/blob/86a41e4ef70ddeca610f2d0ed195a7834fc6bf4d/oci/defaults.go#L116-L134) may be even add a comment to keep these 2 lists in sync with the code in oci/defaults.go ?